event_monitor/src/lib.rs

```diff
@@ -45,17 +81,24 @@ struct Event<'a> {
 
 pub fn event_log(source: &str, event: &str, properties: Option<&HashMap<Cow<str>, Cow<str>>>) {
     // SAFETY: MONITOR is always in a valid state (None or Some).
-    if let Some((file, start)) = unsafe { MONITOR.as_ref() } {
+    if let Some(monitor) = unsafe { MONITOR.as_mut() } {
```

Comment:
Is this thread-safe? I used to be safe as it was read-only for the shared `MONITOR`.

Reply:
It's not, this is actually UB as it's breaking the aliasing rules by creating multiple mutable references, thank you for catching it!

I hastily revised that line in an attempt to make the code more elegant, but I completely overlooked the fact that this will be called from multiple threads, will revise this accordingly.

Reply:
Reverted with the last force-push. Thanks again!